### DISCUSSION TITLE
Possible success with SCIRun 5 install on Ubuntu

### DISCUSSION THREAD

**Discussion by kimjohn1** [SCIInstitute/SCIRun#2239] [General] [answered]
This discussion is somewhat related to Discussion #2232 started 2 weeks ago, same author.
I have installed SCIRun 5 on Ubuntu 18.04 using the steps:
```
sudo apt install cmake g++ qt5-qmake git cmake-curses-gui curl
sudo apt install libgl1-mesa-dev 
```
Then installing Qt5 (version 5.15.2) per the instructions on the Qt website. Then downloading the SCIRun-master.zip file from your website, unpacking the compressed file, navigating to  ~/SCIRun-master/bin and running:
```
cmake -DQt5_PATH=/home/kj/Qt/5.15.2/gcc_64 ../Superbuild
make
cd SCIRun
sudo make install
```
Compile and install steps completed without error.  Can you tell me how I might verify that SCIRun is installed, and how about how to make the program run?

Skeptically,
Kim

**Comment by kimjohn1** [comment, marked as answer]
Issue has been resolved.  Please mark this as closed, or let me know how to do that myself.

**Comment by NeuroMan1** [comment]
Hi @kimjohn1 

I am new to Linux and I was trying to install Scirun from the source files. I faced similar problems to that you discussed in Discussion #2232. But thanks to your notes and comments I managed to install Scirun successfully. However, I have also reached the level where I was wondering how to verify the installation and how to make the program run. Could you please help me with that?
Thanks!

**Reply by kimjohn1** [reply]
Congrats on getting an installation done.  I wrote a SCIRun 5 Basic Tutorial for myself that summarizes what I learned.  It includes step to test an installation by opening the SCIRun application

[SCIRun 5 Basic Tutorial.pdf](https://github.com/SCIInstitute/SCIRun/files/6314997/SCIRun.5.Basic.Tutorial.pdf)


Have a look, and let me know if it is unclear or doesn't work for you.

Kim

**Reply by NeuroMan1** [reply]
Thank you so much for your help. Everything is clear in the tutorial you sent. This helps not only with installation but also with learning the basics. I was able to run SCIRun. 

Thanks again!
Ahmad

**Reply by dcwhite** [reply]
@kimjohn1 Your tutorial is a great resource, can I copy it into the repo for future users? We can add links in various places too. If you have an editable version like in Markdown or something, that might be better for me to commit too, so it can be easily updated.

**Reply by kimjohn1** [reply]
dcwhite

Sure thing.  I wrote it using LibreOffice, so I'll attach .odt and .docx versions.    The current form is mostly lab notes to myself, so edit and use whatever parts or all of it as desired.

Let me know if the attachment doesn't work.

Kim
[Basic_Tutorial.zip](https://github.com/SCIInstitute/SCIRun/files/6338770/Basic_Tutorial.zip)

**Reply by kimjohn1** [reply]
@dcwhite 
I guess I didn't use the correct addressee syntax.   Please see response above.
Kim
